Assets/HoloToolkit/Input/Scripts/BasicCursor.cs

```diff
@@ -28,15 +28,11 @@ public struct RaycastResult
 
         private bool hasLoggedGazeManagerError;
 
+        private GazeManager gazeManagerReference;
```

Comment:
Lets call this gazeManager or gazeManagerInstance to be consistent with other classes.